packages/design/src/DataTable/types.ts

```diff
@@ -31,10 +33,20 @@ export type PaginationConfig = {
 };
 
 export type FetchingConfig = {
-  onFetchMore: () => void;
+  onFetchNext?: () => void;
+  onFetchPrev?: () => void;
+  onFetchMore?: () => void;
   fetchStatus: FetchStatus;
 };
 
+export type ServersideConfig = {
+  // 'from and 'to' are the indices of the first and last items on the page, respectively, for displaying: "SHOWING from - to of totalItemCount"
```

Comment:
What is this comment supposed to apply to? I see a mention of 'from' and 'to' but there's nothing here named 'from' or 'to'.

Reply:
Leftover from a previous implementation that I forgot to delete